### PR TITLE
Fix UploadJob index page ordering

### PR DESCRIPTION
**Pull request by ChrisBr**
and add the created at timestamp to it!

### CODE REVIEW

File: src/api/app/models/cloud/backend/upload_job.rb [hunk 1]
```diff
@@ -42,6 +42,11 @@ def self.all(user)
         []
       end
 
+      def created
+        Time.at(xml_object.created.to_i).to_datetime
+      end
+      alias_method :created_at, :created
```
Comment (bgeuken) [inline src/api/app/models/cloud/backend/upload_job.rb]: Why don't you just overwrite created_at directly?

Comment (ChrisBr) [reply src/api/app/models/cloud/backend/upload_job.rb]: I want to keep that it is possible to access the backend response directly and the backend responds with created.

Comment (bgeuken) [reply src/api/app/models/cloud/backend/upload_job.rb]: Would you mind adding a comment to the commit message explaining why the created method is added and why we don't overwrite created_at?

Comment (ChrisBr) [reply src/api/app/models/cloud/backend/upload_job.rb]: Done!

**Pull request merged**